### ISSUE TITLE
Help with making AS/400 work with the JAWS screen reader

### ISSUE THREAD

**Issue by betsyatatr** [IBMa/equal-access#186]
## Environment

> Operating system: Windows 10

> Browser: Chrome

> Automated testing tool and ruleset:

> Assistive technology used to verify: Fusion 2020 includes the JAWS screen reader

## Detailed description

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes

Information that displays on the screen is not readable by the JAWS 2020 screen reader program.  This is for a blind patient of mine you is seeking employment and this is the only program that is holding her back.  I don't know where else to try.

**Comment by tombrunet** [comment]
@betsyatatr I might be able to track down someone who can help, but I need a little more information. You have a couple of different things here that are a little confusing. I see mention of AS/400, but also JAWS, Windows, and Fusion. Can you describe the environment a bit better?

For instance, you might have a Mac with Fusion that's running Windows with JAWS. I'm having trouble thinking of a combination with AS/400, JAWS, Windows, and Fusion where that software allows installation.

**Comment by tombrunet** [comment]
@betsyatatr Okay, for clarity, I think you're referring to Freedom Scientific's Fusion (https://www.freedomscientific.com/products/software/fusion/) whereas I read it at first as VMWare Fusion, which is a Mac application. I'm still trying to understand which operating system they are accessing first since Windows and AS/400 are two different systems and what they're using to communicate between the two systems. You mention Chrome - are they on a Windows system and trying to access an administrative console on an AS/400 through a browser interface?

**Comment by tombrunet** [comment]
Closing, given no response. Feel free to re-open with more information.
